### PR TITLE
Add a note about version compatibility

### PR DESCRIPTION
**Pull request by pascalberger**
Add note about version / API compatibility

### CODE REVIEW

**Comment by pascalberger** [comment]
@gep13 Do you know if it works like this from the Cake / bindings perspective?

File: input/docs/usage/basic-usage.md [hunk 1]
```diff
@@ -17,6 +17,19 @@ and support for TFS pull requests is imported:
...
 
+:::{.alert .alert-info}
+Please note that the core addin and all issue providers and pull request systems need to be build
+against a compatible core API.
+You can check the required dependencies through the dependencies on the NuGet package of the addins.
+
+This means that it's not possible to use a version of an issue provider build against `Cake.Prca` 1.0.0
+together with a version of a pull request system build against `Cake.Prca` 2.0.0.
+
+We use stric [semantic versioning].
```
Comment (gep13) [inline input/docs/usage/basic-usage.md]: typo - stric -> strict

Comment (gep13) [reply input/docs/usage/basic-usage.md]: Was that meant to be a link to something about semantic versioning?

File: input/docs/usage/basic-usage.md [hunk 2]
```diff
@@ -17,6 +17,19 @@ and support for TFS pull requests is imported:
...
+Therefore it's possible to use issue provider and pull request system built agains `Cake.Prca` 1.0.0
```
Comment (gep13) [inline input/docs/usage/basic-usage.md]: typo - agains -> against

**Review by gep13 — CHANGES_REQUESTED** [review]

**Comment by gep13** [comment]
@pascalberger in theory, yes, I think that this should be fine.  However...

@devlead @patriksvensson @agc93 what are your thoughts?

**Review by gep13 — APPROVED** [review]

**Review by stffabi — APPROVED** [review]

**Pull request merged**